### ISSUE TITLE
Incompatibility with EnhancedRadarBooster

### ISSUE THREAD

**Issue by TheRealMrKam** [dlamkins/LCCollectYourDead#2]
Hello ! I found out that i can't recall my radar boosters anymore because of LCCollectYourDead there is also no recall sound when this happen

**Comment by Miradae** [comment]
Adding some console erros to this bug
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LCCollectYourDead.Patches.ShipTeleporterPatch.CashEmIn (ShipTeleporter __instance) (at <503c92985df94467b184a2921c91d692>:IL_0010)
(wrapper dynamic-method) ShipTeleporter.DMD<ShipTeleporter::beamUpPlayer>(ShipTeleporter)
ShipTeleporter.PressTeleportButtonClientRpc () (at <af9b1eec498a45aebd42601d6ab85015>:IL_0111)
ShipTeleporter.__rpc_handler_2773756087 (Unity.Netcode.NetworkBehaviour target, Unity.Netcode.FastBufferReader reader, Unity.Netcode.__RpcParams rpcParams) (at <af9b1eec498a45aebd42601d6ab85015>:IL_002E)
Unity.Netcode.RpcMessageHelpers.Handle (Unity.Netcode.NetworkContext& context, Unity.Netcode.RpcMetadata& metadata, Unity.Netcode.FastBufferReader& payload, Unity.Netcode.__RpcParams& rpcParams) (at <895801699cfc4b4ab52267f31e2a4998>:IL_004E)
Rethrow as Exception: Unhandled RPC exception!
```

**Comment by Miradae** [comment]
its just a simple nullpointer in
`if (beamedUpPlayer.deadBody != null) {`
which can be fixed with
`if (beamedUpPlayer?.deadBody != null) {`

**Comment by dlamkins** [comment]
Sure is. If somebody wants to PR it I'll gladly accept it or I can do it later tonight and get a new release up.

**Comment by dlamkins** [comment]
Resolved in v1.0.2.
